README.md

```diff
@@ -1 +1,62 @@
-# Portfolio-Project
\ No newline at end of file
+# Portfolio Project
+
+In this project, the header and headline of the mobile view on my portfolio was worked on
+
+# Project Name
+
+**My Portfolio Project**
+
+# Screenshot
+
+## Header Image
+
+![Image of the header](./img/header.png)
+
+## Headline Image
+
+![Image of the headline](./img/headline.png)
+
+
+# Live Website
+
+(https://dammyshittu.github.io/Portfolio-Project/)
+
+# This project was built with:
+
+HTML5
+CSS3
+Linters
+
+# Version Control System
+
+GIT
+
+# Hosting Platform
+
+Github
+
+# Template Source
+
+Figma
+
+# 👤 Author
+
+GitHub: (@dammyShittu)
+
+Twitter: (@aded_shittu)
+
+LinkedIn: (linkedin.com/in/adedamola-shittu-3ab465172/)
```

Comment:
- [ ] Kindly make the links active. Please follow this [link](https://github.com/microverseinc/readme-template) for `README` template.